services/ui-src/src/views/Home/Home.tsx

```diff
@@ -1,73 +1,56 @@
-import { Navigate } from "react-router-dom";
 // components
-import { Box, Flex, Text } from "@chakra-ui/react";
+import { Box, Flex, Heading, Text } from "@chakra-ui/react";
 import { Banner, TemplateCard } from "../../components/index";
 // utils
-import { BannerTypes, UserRoles } from "utils/types/types";
-import { useUser } from "utils/auth";
+import { BannerTypes } from "utils/types/types";
 // data
 import templateCardsVerbiage from "../../data/templateCards.json";
 
-export default () => {
-  const { userRole, userState } = useUser();
-  const isAdminUser = userRole && userRole !== UserRoles.STATE;
-
-  if (isAdminUser) {
-    return <Navigate to={`/admin`} />;
-  }
-
-  return (
-    <section>
-      {userState ? (
-        <Box sx={sx.root} data-testid="home-view">
-          <Banner
-            status={BannerTypes.INFO}
-            bgColor="palette.alt_lightest"
-            accentColor="palette.alt"
-            title="Welcome to the new Managed Care Reporting tool!"
-            description="Each state must submit one report per program."
-          />
-          <Flex sx={sx.mainContentFlex}>
-            <Box sx={sx.leadTextBox}>
-              <Box textStyle="h1" sx={sx.headerText}>
-                Your fiscal year 2022 templates
-              </Box>
-              <Box textStyle="body">
-                Download these templates to begin gathering administrative data
-                for your Medicaid managed care program. Submit your completed
-                report to the Centers for Medicare and Medicaid Services (CMS)
-                through this website beginning October 2022.
-              </Box>
-            </Box>
-            <TemplateCard verbiage={templateCardsVerbiage.MCPAR}></TemplateCard>
-            <TemplateCard verbiage={templateCardsVerbiage.MLR}></TemplateCard>
-            <TemplateCard verbiage={templateCardsVerbiage.NAAAR}></TemplateCard>
-          </Flex>
-        </Box>
-      ) : (
-        <Box data-testid="home-view">
-          <Text>You are not authorized to view this page</Text>
+export default () => (
+  <section>
+    <Box sx={sx.root} data-testid="home-view">
+      <Banner
+        status={BannerTypes.INFO}
+        bgColor="palette.alt_lightest"
+        accentColor="palette.alt"
+        title="Welcome to the new Managed Care Reporting tool!"
+        description="Each state must submit one report per program."
+      />
+      <Flex sx={sx.mainContentFlex}>
+        <Box sx={sx.leadTextBox}>
+          <Heading as="h1" sx={sx.headerText}>
+            Your fiscal year 2022 templates
+          </Heading>
+          <Text as="body">
+            Download these templates to begin gathering administrative data for
+            your Medicaid managed care program. Submit your completed report to
+            the Centers for Medicare and Medicaid Services (CMS) through this
+            website beginning October 2022.
+          </Text>
```

Comment:
Accessibility change here as well. In general we try not to put text directly inside divs because then they act kinda weird sometimes. 

```
<div>
  <p>This is better</p>
</div>
<div>than this.</div>
```